packages/react-components/react-message-bar-preview/stories/MessageBar/Default.stories.tsx

```diff
@@ -1,17 +1,12 @@
 import * as React from 'react';
-import { Button, Link } from '@fluentui/react-components';
-import { DismissRegular } from '@fluentui/react-icons';
-import { MessageBar, MessageBarActions, MessageBarTitle, MessageBarBody } from '@fluentui/react-message-bar-preview';
+import { Link } from '@fluentui/react-components';
+import { MessageBar, MessageBarTitle, MessageBarBody } from '@fluentui/react-message-bar-preview';
 
 export const Default = () => (
   <MessageBar>
     <MessageBarBody>
       <MessageBarTitle>Descriptive title</MessageBarTitle>
       Message providing information to the user with actionable insights. <Link>Link</Link>
     </MessageBarBody>
-    <MessageBarActions containerAction={<Button appearance="transparent" icon={<DismissRegular />} />}>
```

Comment:
nit: The intention in the default example is to showcase all the slots, it's not a requirement for me but rather a nice to have